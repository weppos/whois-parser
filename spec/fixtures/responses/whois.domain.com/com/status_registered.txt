Domain Name: BLUEBUCKWOODWORKING.COM
Domain ID: 2027638587_DOMAIN_COM-VRSN
Registrar WHOIS Server: whois.domain.com
Registrar URL: www.domain.com
Updated Date: 2016-05-13T16:25:54Z
Creation Date: 2016-05-12T09:25:07Z
Registrar Registration Expiration Date: 2017-05-12T09:25:07Z
Sponsoring Registrar: Domain.com, LLC
Sponsoring Registrar IANA ID: 886
Registrar Abuse Contact Email: compliance@domain-inc.net
Registrar Abuse Contact Phone: +1.6027165396
Reseller: FatCow
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Registry Registrant ID: 
Registrant Name: Andreea Luisa
Registrant Organization: bluebuckwoodworking
Registrant Street: Aleea Luptatorilor, nr 32
Registrant City: Bucuresti
Registrant State/Province: NA
Registrant Postal Code: 021447
Registrant Country: RO
Registrant Phone: +40.0744222769
Registrant Phone Ext: 
Registrant Fax: 
Registrant Fax Ext: 
Registrant Email: bluebuckwood@mail.com
Registry Admin ID: 
Admin Name: Andreea Luisa
Admin Organization: bluebuckwoodworking
Admin Street: Aleea Luptatorilor, nr 32
Admin City: Bucuresti
Admin State/Province: NA
Admin Postal Code: 021447
Admin Country: RO
Admin Phone: +40.0744222769
Admin Phone Ext: 
Admin Fax: 
Admin Fax Ext: 
Admin Email: bluebuckwood@mail.com
Registry Tech ID: 
Tech Name: Andreea Luisa
Tech Organization: bluebuckwoodworking
Tech Street: Aleea Luptatorilor, nr 32
Tech City: Bucuresti
Tech State/Province: NA
Tech Postal Code: 021447
Tech Country: RO
Tech Phone: +40.0744222769
Tech Phone Ext: 
Tech Fax: 
Tech Fax Ext: 
Tech Email: bluebuckwood@mail.com
Name Server: NS1.HOST-CARE.COM
Name Server: NS2.HOST-CARE.COM
DNSSEC: unsigned
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
>>> Last update of WHOIS database: 2016-05-13T16:25:54Z <<<

"For more information on Whois status codes, please visit https://icann.org/epp"

Registration Service Provider:
    FatCow, support@fatcow.com
    888-278-9780


