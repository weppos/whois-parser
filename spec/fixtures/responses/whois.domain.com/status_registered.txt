Domain Name: DOMAIN.COM
Domain ID: 608082_DOMAIN_COM-VRSN
Registrar WHOIS Server: whois.domain.com
Registrar URL: www.domain.com
Updated Date: 2016-10-21T20:11:05Z
Creation Date: 1994-07-01T04:00:00Z
Registrar Registration Expiration Date: 2021-01-07T13:34:24Z
Registrar: Domain.com, LLC
Registrar IANA ID: 886
Reseller: Domain Name Holding Company, Inc
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Registry Registrant ID: 
Registrant Name: Domain Administrator
Registrant Organization: Endurance International Group West, Inc
Registrant Street: 10 Corporate Drive Suite 300
Registrant City: Burlington
Registrant State/Province: MA
Registrant Postal Code: 01803
Registrant Country: US
Registrant Phone: +1.3604495900
Registrant Phone Ext: 
Registrant Fax: +1.3602534234
Registrant Fax Ext: 
Registrant Email: corpdomains@endurance.com
Registry Admin ID: 
Admin Name: Domain Administrator
Admin Organization: Endurance International Group West, Inc
Admin Street: 10 Corporate Drive Suite 300
Admin City: Burlington
Admin State/Province: MA
Admin Postal Code: 01803
Admin Country: US
Admin Phone: +1.3604495900
Admin Phone Ext: 
Admin Fax: +1.3602534234
Admin Fax Ext: 
Admin Email: corpdomains@endurance.com
Registry Tech ID: 
Tech Name: Domain Administrator
Tech Organization: Endurance International Group West, Inc
Tech Street: 10 Corporate Drive Suite 300
Tech City: Burlington
Tech State/Province: MA
Tech Postal Code: 01803
Tech Country: US
Tech Phone: +1.3604495900
Tech Phone Ext: 
Tech Fax: +1.3602534234
Tech Fax Ext: 
Tech Email: corpdomains@endurance.com
Name Server: NS-166.AWSDNS-20.COM
Name Server: NS-683.AWSDNS-21.NET
Name Server: NS-1250.AWSDNS-28.ORG
Name Server: NS-2022.AWSDNS-60.CO.UK
DNSSEC: unsigned
Registrar Abuse Contact Email: compliance@domain-inc.net
Registrar Abuse Contact Phone: +1.6027165396
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
>>> Last update of WHOIS database: 2016-10-21T20:11:05Z <<<

"For more information on Whois status codes, please visit https://icann.org/epp"

Registration Service Provider:
    Domain Name Holding Company, Inc, corpdomains@endurance.com
    +1.8007972958
    This company may be contacted for domain login/passwords,
    DNS/Nameserver changes, and general domain support questions.

