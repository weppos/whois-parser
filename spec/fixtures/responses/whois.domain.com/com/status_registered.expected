#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2016-05-12T09:25:07Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2016-05-13T16:25:54Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2017-05-12T09:25:07Z}


#registrar
  %s %CLASS{registrar}
  %s.id           == "886"
  %s.name         == "Domain.com, LLC"
  %s.organization == "Domain.com, LLC"
  %s.url          == "www.domain.com"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "Andreea Luisa"
  %s[0].organization  == "bluebuckwoodworking"
  %s[0].address       == "Aleea Luptatorilor, nr 32"
  %s[0].city          == "Bucuresti"
  %s[0].zip           == "021447"
  %s[0].state         == "NA"
  %s[0].country       == nil
  %s[0].country_code  == "RO"
  %s[0].phone         == "+40.0744222769"
  %s[0].email         == "bluebuckwood@mail.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == nil
  %s[0].name          == "Andreea Luisa"
  %s[0].organization  == "bluebuckwoodworking"
  %s[0].address       == "Aleea Luptatorilor, nr 32"
  %s[0].city          == "Bucuresti"
  %s[0].zip           == "021447"
  %s[0].state         == "NA"
  %s[0].country       == nil
  %s[0].country_code  == "RO"
  %s[0].phone         == "+40.0744222769"
  %s[0].email         == "bluebuckwood@mail.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "Andreea Luisa"
  %s[0].organization  == "bluebuckwoodworking"
  %s[0].address       == "Aleea Luptatorilor, nr 32"
  %s[0].city          == "Bucuresti"
  %s[0].zip           == "021447"
  %s[0].state         == "NA"
  %s[0].country       == nil
  %s[0].country_code  == "RO"
  %s[0].phone         == "+40.0744222769"
  %s[0].email         == "bluebuckwood@mail.com"


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.host-care.com"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.host-care.com"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
